### ISSUE TITLE
Improve theming support of BB (Tracking Issue)

### ISSUE THREAD

**Issue by Tobias-B-Besemer** [bleachbit/wishlist#75]
Would be nice if you can at any point improve the theming support of BB!

This can maybe be a tracking issue for some ideas to it...

Think the first issue to it would be, to remove the "Dark Mode" radio button again in the settings, add therefor a new tab there, show in the tab a list of found themes (folders) and let the user select one. "Dark Mode" setting from config file should be by first run of new BB version be used to set the theme want!

When themes can (manual) be added to BB by copy theme folder into prog sub-dirs, I will talk to some very talented GFX artists I know if they want to create first themes that can be used... I will also try to create a theme for BB...

**Comment by Tobias-B-Besemer** [comment]
Btw.: I guess you have GIMP/Photoshop/etc. files of the themes, you haven't released to the public, yet...
Those files would help people like me who wanna build based on a existing one...
So it would be nice, if you e.g. make a new repo for the files ("bleachbit-themes" ???) and release the files there.

**Comment by az0** [comment]
>Btw.: I guess you have GIMP/Photoshop/etc. files of the themes, you haven't released to the public, yet...

No, I don't have anything like this. I use the standard, unmodified themes.

**Comment by Tobias-B-Besemer** [comment]
And e.g. the icons for preview and clean and etc. ... No GFX Working Files ???

**Comment by az0** [comment]
>And e.g. the icons for preview and clean and etc. ... No GFX Working Files ???

They're the standard "named icons" that come with the system. For example, the preview button is the icon called **edit-find**.

You can find more info here

https://specifications.freedesktop.org/icon-naming-spec/latest/ar01s04.html

https://developer.gnome.org/gtk3/stable/gtk3-icon-browser.html

https://lazka.github.io/pgi-docs/Gtk-3.0/classes/Image.html#Gtk.Image.new_from_icon_name

**Comment by Tobias-B-Besemer** [comment]
Thx! :)

**Comment by az0** [comment]
In BleachBit 4.0 we have dark mode and Windows 10 theme. This issue would extend those efforts to more themes. On Windows, the list of themes is fixed, while on Linux, there can be many themes.

GTK distinguishes themes and icon themes, and a problem is some [icon themes apparently don't have all the stock icons](https://github.com/bleachbit/bleachbit/issues/799) that BleachBit expects.
